src/App.css

```diff
@@ -28,44 +28,90 @@
   to { transform: rotate(360deg); }
 }
 
+.ambassadors {
+  display: flex;
+  flex-flow: row wrap;
+  /* width: 85%; */
+  height: 100px;
+  -webkit-margin-before: 1em;
+  -webkit-margin-after: 1em;
+  -webkit-margin-start: 0px;
+  -webkit-margin-end: 0px;
+  -webkit-padding-start: 40px
+}
+
+.ambassador-lists {
+  margin: 8px;
+}
+
+#ambassador-image {
+  border: 5px solid cadetblue;
+  border-radius: 50%;
+}
+
 .profile-cards {
   margin: 10px;
-  padding: 10px;
-  width: 200px;
+  width: 220px;
   background: rgb(220, 225, 223);
-  box-shadow:  0.5px 0.5px 0.5px 0.5px grey;
+  /* box-shadow:  0.5px 0.5px 0.5px 0.5px grey; */
 }
 
-.profile-image {
-  border-bottom: 1px solid white;
-  width: 200px;
-  margin-bottom: 10px;
-  overflow: hidden;
+.info-header{
+  padding: 3px 0 0 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: row wrap;
+  background-color: rgba(255, 255, 255, 0.8);
 }
 
-.profile-info{
+.social-icon-container {
   display: flex;
-  align-items: center; 
-  width: 200px;
-  height: 30px;
-  margin-bottom: 10px;
+  padding-right: 5px;
+}
+
+.social-icons {
+  margin:0 3px 0 3px;
+}
+
+.info-wrapper{
+  position: absolute;
+  top: 190px;
+  width: 220px;
+  background-image: url("./_images/cards_background.png");
+}
+
+.info-container{
+  margin: 10px;
+  margin-top: 30px;
 }
 
 .profile-name {
-  font-size: 13px;
+  color: black;
+  font-size: 14px;
   font-weight: 700;
-  margin-right: 5px;
+  padding: 0 8px 0 8px;
 }
 
-.nav-symbols {
-  width: 20px;
-  height: 20px;
-  margin: 1px;
-}
 
 .profile-description {
   margin-bottom: 10px;
+  padding: 10px 8px 8px 8px;
+}
+
+#description {
```

Comment:
Is there any reason description is an id and not a class. In general the only times to use id's is when the item should be find able with a #link. eg https://github.com/facebook/create-react-app#whats-included

Reply:
Forgot why I did that. I have removed now and moved the styling into `profile-description`. :)